### PR TITLE
chore: cache cni v1.2.0

### PR DESCRIPTION
**Pull request by thatmattlong**

### CODE REVIEW

**Comment by thatmattlong** [comment]
Relevant aks-e pr: https://github.com/Azure/aks-engine/pull/4058

**Comment by paulgmiller** [comment]
Jack is changing the CSE in aks-engien to make transparent the default. We might want to mirror the same changes here.

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -97,10 +97,9 @@ cat << EOF >> ${VHD_LOGS_FILEPATH}
 EOF
 
 VNET_CNI_VERSIONS="
+1.2.0
 1.1.8
-1.1.7
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:101]: just to confirm 1.1.7 is not in use (or will not be in use by the time this VHD is released?)

Comment (paulgmiller) [reply vhdbuilder/packer/install-dependencies.sh:101]: Yes we finished toggle release to 1.1.8 two weeks ago.

**Pull request merged**